lib/Backend.hs

```diff
@@ -20,8 +20,10 @@ import Polysemy.Error                (Error)
 import Polysemy
 import Coffer.Path (EntryPath, Path)
 import BackendName (BackendName)
+import Network.HTTP.Client (Manager)
+import Polysemy.State (State)
 
-type Effects r = (Member (Embed IO) r, Member (Error CofferError) r)
+type Effects r = (Member (Embed IO) r, Member (State (Maybe Manager)) r, Member (Error CofferError) r)
```

Comment:
Using `State` will work here for now, but `State` is meant for single-threaded applications and we'll get into trouble when we introduce parallelism in #4.

I think it's simpler to just create two `Manager`s when the app starts.

Reply:
What about using `Reader` then?

Reply:
> What about using Reader instead of State?

Yup, sounds good to me!